jwst/datamodels/slit.py

```diff
@@ -36,6 +36,8 @@ def __init__(self, init=None, data=None, dq=None, err=None,
                  relsens=None, area=None,
                  wavelength_pointsource=None, pathloss_pointsource=None,
                  wavelength_uniformsource=None, pathloss_uniformsource=None,
+                 wavelength_pointsource2d=None, pathloss_pointsource2d=None,
```

Comment:
There aren't actually wavelength_pointsource2d or wavelength_uniformsource2d attributes, the regular wavelength attribute is the corresponding wavelength values for each of the pathloss2d attributes.